cmd/fluxctl/root_cmd.go

```diff
@@ -56,6 +58,7 @@ const (
 	envVariableToken      = "FLUX_SERVICE_TOKEN"
 	envVariableCloudToken = "WEAVE_CLOUD_TOKEN"
 	defaultURLGivenToken  = "https://cloud.weave.works/api/flux"
+	envVariableTimeout    = "FLUX_TIMEOUT"
```

Comment:
I prefer these sorted by prefix (all `env*` constants below each other) but I may have OCD.

Reply:
done, I took the liberty of sorting the entire thing 😆